linux/runtime.go

```diff
@@ -67,6 +67,8 @@ type Config struct {
 	Runtime string `toml:"runtime,omitempty"`
 	// NoShim calls runc directly from within the pkg
 	NoShim bool `toml:"no_shim,omitempty"`
+	// Debug enable debug on the shim
```

Comment:
Should add a note, somehow, in logs that when shim debugging is enabled, if contianerd dies, the shims will also die because of the io being closed